### PR TITLE
practice file 

### PR DESCRIPTION
**Pull request by sriram-techie**
this is just for practice please approve.

### CODE REVIEW

File: practice.tf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+this is just for practice
```
Comment (sriram182000) [inline practice.tf:1]: why is this line added ? Is this necessary ?

Comment (sriram-techie) [reply practice.tf:1]: Yes it is needed

**Review by sriram182000 — APPROVED** [review]
Everything seems good to me. Go ahead. Approving it.
